pkg/security/ebpf/c/include/structs/filesystem.h

```diff
@@ -20,14 +20,10 @@ struct mount_fields_t {
     dev_t device;
     u32 bind_src_mount_id;
     char fstype[FSTYPE_LEN];
-};
-
-struct fsmount_fields_t {
-    struct path_key_t root_key;
-    s32 fd;
-    u32 flags;
-    dev_t device;
-    unsigned int mount_attrs;
+    struct {
+        u32   visible :  1;  // Is mount visible in the VFS?
+        u32   detached : 1;  // A detached mount is always not visible, but an invisible mount isn't always detached
+    } params;
```

Comment:
not a huge fan of this, at the end we can't use an u8 to gain memory and have to stick to a u32, so saving those in like two u16 will be much simpler right?